packages/cluster-ui/src/tableStatistics/tableStatistics.tsx

```diff
@@ -5,8 +5,14 @@ import { statisticsClasses } from "../transactionsPage/transactionsPageClasses";
 import { ISortedTablePagination } from "../sortedtable";
 import { Button } from "src/button";
 import { ResultsPerPageLabel } from "src/pagination";
+import { Tooltip } from "../tooltip";
```

Comment:
Just a head's up, we are consolidating tooltips to use the `Tooltip` from `@cockroachlabs/ui-components`. See #251.

@elkmaster, I was looking at this PR to see how replacing the `Tooltip` component might work and ran into two things,
1. We don't extend `React.HTMLAttributes<HTMLDivElement>` props in Tooltip (or any html attribute props) and so passing a `className` prop won't be applied.
2. I tried adding `Tooltip` here in the PR and got some `z-index` issues. Any ideas?
<img width="550" alt="Screen Shot 2021-04-07 at 15 36 09" src="https://user-images.githubusercontent.com/397448/113923802-fbd7d480-97b6-11eb-86e4-ec450af42b04.png">

Reply:
In my pr z-index issue is fixed. Regarding classname, in this specific case it's just "float: right". I am sure we can handle it without passing inside tooltip. Lets merge this pr first and I will handle this all on my side.